utility/src/main/java/org/oppia/android/util/parser/html/LiTagHandler.kt

```diff
@@ -23,7 +24,7 @@ const val CUSTOM_LIST_OL_TAG = "oppia-ol"
 class LiTagHandler(
   private val context: Context,
   private val displayLocale: OppiaLocale.DisplayLocale
-) : CustomHtmlContentHandler.CustomTagHandler {
+) : CustomHtmlContentHandler.CustomTagHandler, CustomHtmlContentHandler.ContentDescriptionProvider {
```

Comment:
Since `getContentDescription` doesn't do anything here, I don't think
`CustomHtmlContentHandler.ContentDescriptionProvider` should be implemented here.